enable-azure-hybrid-benfit-intent/arm template and bicep/README.md

```diff
@@ -0,0 +1,17 @@
+# AzSHCI-Automation
+
+Manage Azure Hybrid Benefit settings for Azure Stack HCI clusters using PowerShell and Azure CLI.
+
+Pre-requisites:
+	1. Create Workspace 'microsoft.operationalinsights/workspaces' manually before running script.
```

Comment:
from where did this come?

Reply:
There was a similar Readme file for ama installation, so to have consistency I added these. If they are unnecessary I can remove them.